PageIndex.md

```diff
@@ -17,11 +17,13 @@
   - under the License.
   -->
 
-# ColumnIndex Layout to Support Page Skipping
+# Parquet page index: Layout to Support Page Skipping
 
-This document describes the format for column index pages in the Parquet
-footer. These pages contain statistics for DataPages and can be used to skip
-pages when scanning data in ordered and unordered columns.
+In Parquet, a *page index* is optional metadata for a
+ColumnChunk, containing statistics for DataPages that can be used
+to skip those pages when scanning in ordered and unordered columns.
```

Comment:
With statistics enabled, only OffsetIndex is presented, with can be used to planning IO and get first_row_ordinal for ColumnChunk?

Reply:
I am sorry @mapleFU  -- I don't quite follow this suggestion

Do you mean that some writers may write only `OffsetIndex` and that both structures may not be present?

Reply:
Yes, ColumnIndex will not be created when there is no valid min/max statistics. In parquet-cpp, we can also disable creating ColumnIndex but enabling OffsetIndex.

Reply:
I added 

```diff
--- a/src/main/thrift/parquet.thrift
+++ b/src/main/thrift/parquet.thrift
@@ -1005,6 +1005,8 @@ struct PageLocation {
  * Optional offsets for each data page in a ColumnChunk.
  *
  * Forms part of the page index, along with ColumnIndex.
+ *
+ * OffsetIndex may be present even if ColumnIndex is not.
  */
 struct OffsetIndex {
   /**
@@ -1026,6 +1028,8 @@ struct OffsetIndex {
  *
  * Forms part the page index, along with OffsetIndex.
  *
+ * If this structure is present, OffsetIndex must also be present.
+ *
  * For each field in this structure, <field>[i] refers to the page at
  * OffsetIndex.page_locations[i]
  */
```

in 8d59c7d to try and clarify this